src/main/lighting/DirectionalLight.java

```diff
@@ -0,0 +1,43 @@
+package lighting;
+
+import primitives.Colour;
+import primitives.NormalizedVector;
+import primitives.Point;
+
+/**
+ * Abstract base class for all lights which have a source.
+ *
+ * @author Abraham Murciano
+ * @author Eli Levin
+ */
+public class DirectionalLight extends LightSource {
+
+	private NormalizedVector direction;
+
+	public DirectionalLight(Colour colour, NormalizedVector direction) {
+		super(colour);
+		this.direction = direction;
+	}
+
+	/**
+	 * Calculate the colour of the light that this light source is projecting onto a given point.
+	 *
+	 * @param point The point at which to calculate the colour of the light.
+	 * @return The colour of the light at the given point.
+	 */
```

Comment:
btw u dont need javadoc for overriden comments, unless you wanna change the documentation for the overridden method